### PR TITLE
added NetworkView

### PR DESCRIPTION
**Pull request by TamarAuf**

### CODE REVIEW

File: src/main/java/NodeView.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            x = (int) (networkCenterX + networkRadius * Math.cos(angle));
+            y = (int) (networkCenterY + networkRadius * Math.sin(angle));
+            xyValues[0] = x;
+            xyValues[1] = y;
+            nodeValues.put(listNodes.get(i), xyValues);
+            g.drawOval(x, y, nodeSize, nodeSize);
+        }
+    }
+
+    void paintConnections(Graphics g) {
```
Comment (TamarAuf) [inline src/main/java/NodeView.java]: The connections are not being drawn

File: src/main/java/NodeView.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.HashMap;
+
+public class NodeView extends JComponent {
+
+    private final Graph graph;
+    private final int nodeSize = 20;
+    private int networkRadius = 0;
+    private int viewSize = networkRadius + 300;
```
Comment (TamarAuf) [inline src/main/java/NodeView.java]: The size is messed up with five nodes

Comment (lliebman) [reply src/main/java/NodeView.java]: Like when there's specifically five nodes something looks wrong?

File: src/main/java/NodeView.java (hunk 1 above)
Comment (lliebman) [reply src/main/java/NodeView.java]: do you know what part of this is broken?

Comment (TamarAuf) [reply src/main/java/NodeView.java]: No, but it's probably either this: "g.drawLine(nodeValues.get(node)[0], nodeValues.get(node)[1], nodeValues.get(connection)[0], nodeValues.get(connection)[1])" or the for loops.

File: src/main/java/NodeView.java (hunk 2 above)
Comment (TamarAuf) [reply src/main/java/NodeView.java]: Yes. And 8 looks weird too.

**Pull request merged**